index.js

```diff
@@ -38,15 +38,20 @@ const ESCAPE_CODES = new Map([
 const wrapAnsi = code => `${ESCAPES[0]}[${code}m`;
 
 module.exports = (str, begin, end) => {
-	end = end || str.length;
+	const arr = Array.from(str.normalize());
+
+	end = end || arr.length;
```

Comment:
Oh, you meant in regards to `||` (need sleep, lol). Yeah, we might have to fix that.